qt/scientific_interfaces/Indirect/IndirectFitPropertyBrowser.cpp

```diff
@@ -117,20 +117,24 @@ QStringList IndirectFitPropertyBrowser::getLocalParameters() const {
 void IndirectFitPropertyBrowser::syncFullBrowserWithTemplate() {
   auto const fun = m_templateBrowser->getFunction();
   if (fun) {
+    m_functionBrowser->blockSignals(true);
```

Comment:
If any of `setFunction`, `updateMultiDataSetParamters`, `setGlobalParameters` or `setCurrentDataSe` throw our `functionBrowser` will be left with blocked signals.

Reply:
Good thinking. and a good solution too. I'll sort that